src/modules/endpoint/models/CategoriesModel.js

```diff
@@ -0,0 +1,79 @@
+import normalizeUrl from 'normalize-url'
+
+/**
+ * Contains a Map [string -> CategoryModel] and some helper functions
+ */
+class CategoriesModel {
```

Comment:
I think we agreed on a Map? @maxammann is implementing some changes in the cms, so that availableLanguages and parent is also returned as urls instead of the ids that are used right now. So if these changes are applied, we also have direct url mapping there. converting the map to an array will then only be necessary anymore for getting all children, so imo the Map is the right implementation. Regarding the naming I agree, CategoriesContainer would be more appropriate

Reply:
Afaik the url is only planned for the availableLanguages-array, but that doesn't affect our data structure, this only avoids having to redirect in order to load the payload in another language with the right ids.
But -> slack